frontend/ts-src/lib/utils.ts

```diff
@@ -0,0 +1,176 @@
+/* Utility functions. */
+
+
+export function isEmail(email: string): boolean {
```

Comment:
Why not just use https://github.com/validatorjs/validator.js instead of re-inventing the wheel?

Reply:
- Didn't know that existed
- Most of the actual validation is being done on the backend and I just wanted a sanity check.
- Just had a quick look, seems like a huge over kill and an extra dependancy for something that will get handled on the backend anyways (I dont think its useful to do thorough email validation on the client)